database/migrations/20190731130001_itinerary.js

```diff
@@ -0,0 +1,51 @@
+exports.up = knex => knex.schema.createTable('itinerary', (table) => {
+  table.increments('id');
+  table.integer('user_id').notNullable()
```

Comment:
Could probably move the chained notNullable call to a new line here for consistency.